gradlew

```diff
@@ -84,6 +84,10 @@ Please set the JAVA_HOME variable in your environment to match the
 location of your Java installation."
 fi
 
+# Create the neccessary templates for IPython
+mkdir ~/.ipython/kernels/kotlin
```

Comment:
"mkdir -p" would be better cause in my case I don't even have the kernels folder.